app/views/pets/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>All pets</h1>
+
+<% @pets.each do |pet| %>
+  <img src="<%= pet.image %>" alt-"it could be yours" width = "300" height = "300"</p>
```

Comment:
Check on this with instructor tomorrow and make an update on the comment here...